### DISCUSSION TITLE
Ability to turn on buffered output

### DISCUSSION THREAD

**Discussion by coreone** [CleanCut/green#189] [Enhancements]
I was trying to use some sample code from the following:

https://stackoverflow.com/questions/4219717/how-to-assert-output-with-nosetest-unittest-in-python/31281467

The code in question is as follows and is supposed to be usable within a test method:

```python
if not hasattr(sys.stdout, "getvalue"):
    self.fail("Test needs to run in buffered mode")
output = sys.stdout.getvalue().strip()
```

This fails with green because I cannot turn on the buffer option of unittest2 with `-b, --buffer`    to "Buffer stdout and stderr during tests".  The following without green works:

`python -m unittest2 -bv -c`

Is there any way to add a flag to green such that it can turn on the buffer feature unittest2 already has built into it?  Is there a way to do this with green already that I have just not been able to figure out?

**Comment by CleanCut** [comment]
I'm not familiar with the buffered output option!  I'll have to check it out.
